vvenancio297/library
# SETUP: Create project structure

# Description

It creates the rails application. It also adds some dependencies like:
- dotenv
- rspec
- pry

## How to test it

Download this branch into your computer with this command `git fetch --all` and then, switch to this branch running `git checkout feature/project-setup`. To check if everything went good start the server running `rails s`
